server/schema/tags/createTag.js

```diff
@@ -0,0 +1,26 @@
+const schema = {
+  required: ['name', 'description'],
+  properties: {
+    name: {
+      type: 'string',
+      minLength: 4,
```

Comment:
I think this is not required

Reply:
why?

Reply:
Maybe 3 but not 4, I think a tag could exist with length 3

Reply:
Mark it as 2. Like some tags can be 'dp', 'dfs', etc.

Reply:
Alright I guess this is okay, let us allow make `minLength` = 4